app/main.py

```
@@ -0,0 +1,197 @@
+import sys
+from typing import Dict, List, Optional, Tuple
+
+import uvicorn
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.openapi.utils import get_openapi
+
+from app.routes import router as app_router
+from app.websockets import websocket_router
+from meetingbaas_pipecat.utils.logger import configure_logger
+from utils.ngrok import LOCAL_DEV_MODE, NGROK_URL_INDEX, NGROK_URLS, load_ngrok_urls
```

Ruff (0.8.2)

`utils.ngrok.LOCAL_DEV_MODE` imported but unused

Remove unused import

(F401)